src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -46,18 +46,28 @@ <h5>{{ version }}</h5>
              i18n-aria-label
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
-        <ng-template #healthChecks>
-          <cd-health-checks [healthData]="healthData.health.checks"></cd-health-checks>
-        </ng-template>
+
         <ng-template #healthWarningAndError>
-        <div class="info-card-content-clickable mt-1"
+        <!-- <div > -->
+          <cds-toggletip
              [ngStyle]="healthData.health.status | healthColor"
-             [ngbPopover]="healthChecks"
+             [isOpen]="isOpen"
+             [dropShadow]="true"
+             [caret]="true"
+             placement="bottom"
+             class="info-card-content-clickable mt-1"
              popoverClass="info-card-popover-cluster-status">
```

Comment:
let's remove this class. I think that was for the previous bootstrap style
```suggestion
```

we could also remove https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card-popover.scss, since the tooltip is taking care of it now?